### DISCUSSION TITLE
🌳XSOSI 计划 8 月份数据

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#173] [2️⃣ 聊项目]
接 XSOSI 计划 [6 月份数据](https://github.com/orgs/X-lab2017/discussions/115)、[7 月份的数据](https://github.com/X-lab2017/open-wonderland/discussions/149)，8 月份的数据如下：

@frank-zsy 可以在下面补充~

**Comment by frank-zsy** [comment]
<img width="1248" alt="image" src="https://user-images.githubusercontent.com/8512426/187879868-84a7e441-92c4-4dcd-b339-c2504a4bda2c.png">
<img width="1308" alt="image" src="https://user-images.githubusercontent.com/8512426/187879896-c7d1b443-8567-4e71-bfa0-205fb560225a.png">
<img width="1320" alt="image" src="https://user-images.githubusercontent.com/8512426/187879901-95bb1b1d-5dc0-478f-9796-30c68b32c2ff.png">
实验室总体 OpenRank 8 月份下降了 9%，与暑假略有关系~
